### PR TITLE
https://github.com/stolostron/rhacm-docs/pull/4377 -- adding info for…

### PR DESCRIPTION
**Pull request by swopebe**
… 2.8 changes

### CODE REVIEW

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 1]
```diff
@@ -59,9 +59,9 @@ To specify the details about running the Ansible template while creating a templ
 
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
-. Select or enter the name of the prehook and posthook Ansible jobs to add to the installation or upgrade of the cluster. 
-+
-*Note:* The _Automation template name_ must match the name of the Ansible job on the {aap-short}.
+. Select between a `Ansible job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags` and workflow types.
```
Comment (Randy424) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]:
```suggestion
. Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags` and workflow types.
```

Comment (Randy424) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: See screenshot...
Screenshot 2023-06-12 at 12 28 09 PM

**Review by Randy424 — APPROVED** [review]
/lgtm

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 2]
```diff
@@ -59,9 +59,9 @@ To specify the details about running the Ansible template while creating a templ
 
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
-. Select or enter the name of the prehook and posthook Ansible jobs to add to the installation or upgrade of the cluster. 
-+
-*Note:* The _Automation template name_ must match the name of the Ansible job on the {aap-short}.
+. Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags` and workflow types.
```
Comment (oafischer) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]:
```suggestion
. Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
```

**Review by oafischer — APPROVED** [review]
1 suggestion, otherwise looks great

**Comment by swopebe** [comment]
Yay @oafischer thank you so much!

**Pull request merged**